modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -112,10 +125,55 @@ ifdef::upi[]
 
 . Update the `storageProfile.imageReference` field by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters by using the values from your offer. You can find a sample template in the "Creating additional worker machines in Azure" section.
 endif::upi[]
+ifdef::mapi[]
+. Record the image details of your offer, specifically the values for `publisher`, `offer`, `sku`, and `version`.
+endif::mapi[]
+
+ifdef::ipi[]
+.Sample `install-config.yaml` file with the Azure Marketplace worker nodes
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    azure:
+      type: Standard_D4s_v5
+      osImage:
+        publisher: redhat
+        offer: rh-ocp-worker
+        sku: rh-ocp-worker
+        version: 4.8.2021122100
+  replicas: 3
+----
+endif::ipi[]
```

Comment:
@bscott-rh I am still working on this (need to look into build fail), but is there a reason your [backport](https://github.com/openshift/openshift-docs/pull/51370/files#diff-ef88389773a21f05d05594b4f02df001e38096a19152390bf9a73c6b5135a6db) doesn't include this for IPI?

Reply:
wheeeee thanks, I will ping you on Slack